pages/view/_iscnId.vue

```diff
@@ -332,14 +339,21 @@
                 :label="$t('iscn.meta.creator.name')"
                 class="w-[50%] my-[12px]"
               >
-                <Label :text="stakeholderInfo.authorName" tag="div" preset="p4" />
+                <Link
+                  v-if="stakeholderInfo.authorWalletAddresses.length"
+                  :class="'text-[16px]'"
+                  :href="`/search?q=${stakeholderInfo.authorWalletAddresses[0].address}`" 
```

Comment:
can we use name route instead of URL, and use explicit query for each field instead of query All (q)?